cmd/sponged/main.go

```diff
@@ -48,7 +48,7 @@ func main() {
 		writeTimeout = 30 * time.Second
 	)
 
-	if err := app.Run(":4001", routes.API(), readTimeout, writeTimeout); err != nil {
+	if err := app.Run(":", routes.API(), readTimeout, writeTimeout); err != nil {
```

Comment:
haha! yes. Let me put it back. I was just checking that it was not really use it somehow 👎